schemas/repositories.json

```diff
@@ -7,7 +7,7 @@
     "properties": {
       "app_id": {
         "type": "string",
-        "format": "^[a-zA-Z][a-zA-Z-]*$"
```

Comment:
Is this something we should enforce in the SDK as well?

Reply:
I don't think so. App id is transformed in the pipeline so it would be different constraints. What's the likelihood of someone requiring an app id that doesn't match those requirements?

Reply:
I'm not aware of any requirements -- these specific ids in these changes came from iOS where maybe upper case letters are a thing (they aren't on Android), but I doubt they are case-sensitive -- even if they were, Mozilla controls the names so it's very unlikely that differences only in case would matter.